### PR TITLE
Port: Changes to cruciform energy regeneration and rituals. (#5975)

### PR DESCRIPTION
**Pull request by Jamini**
## About The Pull Request
Ports Eris #5975
Tweaks 5975 heavily to match Occulus lore

Cruciform energy regeneration is now based of COG, increasing at 0.01 per point up to 0.75
Base cruciform energy regeneration reduced to 25 from 50
Pain stutter no longer blocks entreaty
revelation restored up to 10 sanity up from 5
Short boost litanies can once again target the user

## Why It's Good For The Game

Tis a port
Tis a port that cut out the shitty eris bits

## Changelog
```changelog
balance: Cruciform energy regeneration is now based of COG, increasing at 0.01 per point up to 0.75
balance: Base cruciform energy regeneration reduced to 25 from 50
feature: Pain stutter no longer blocks entreaty
balance: revelation restored up to 10 sanity up from 5
tweak: Short boost litanies can once again target the user
```

### CODE REVIEW

File: code/modules/core_implant/cruciform/rituals/machinery.dm [hunk 1]
```diff
@@ -30,7 +30,18 @@
 		fail("Cloner is off.", user, C)
 		return FALSE
 
+<<<<<<< HEAD
```
Comment (Michiyamenotehifunana) [inline code/modules/core_implant/cruciform/rituals/machinery.dm:33]: Uhhh, is this supposed to be here

File: code/modules/core_implant/cruciform/rituals/machinery.dm [hunk 2]
```diff
@@ -30,7 +30,18 @@
...
+=======
+	if(pod.start())
+		var/damage_modifier = 1
+		var/obj/item/weapon/implant/core_implant/cruciform/C_user = user.get_core_implant(/obj/item/weapon/implant/core_implant/cruciform)
+		if(C_user.get_module(CRUCIFORM_INQUISITOR) || (C_user.get_module(CRUCIFORM_PRIEST) && C_user.get_module(CRUCIFORM_REDLIGHT)))
+			damage_modifier = 0
+		else if(C_user.get_module(CRUCIFORM_PRIEST))
+			damage_modifier = 0.5
+		pod.clone_damage = clone_damage * damage_modifier
+>>>>>>> f79716f7f... Changes to cruciform energy regeneration and rituals. (#5975)
```
Comment (Michiyamenotehifunana) [inline code/modules/core_implant/cruciform/rituals/machinery.dm:44]: I think you forgot to fix a conflict

Comment (Jamini) [reply code/modules/core_implant/cruciform/rituals/machinery.dm:44]: This file is disabled. We don't use these litanies.

**Review by Michiyamenotehifunana — APPROVED** [review]
Looks about right.

**Pull request merged**